src/sentry/workflow_engine/models/workflow_action_group_status.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class WorkflowActionGroupStatus(DefaultFieldsModel):
+    """
+    Stores when a workflow action last fired for a Group.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
If an account is relocated, do we not want the history of their alert workflows to be preserved?

Reply:
hmm this is the workflow engine equivalent to `GroupRuleStatus` which is excluded from relocation. i'm not entirely sure why, though

Reply:
Yeah this is much too much data to relocate, we usually exclude things that are related to groups